source.go

```diff
@@ -23,6 +23,7 @@ import (
 	"github.com/conduitio/conduit-commons/csync"
 	"github.com/conduitio/conduit-connector-postgres/source"
 	"github.com/conduitio/conduit-connector-postgres/source/logrepl"
+	cpool "github.com/conduitio/conduit-connector-postgres/source/pool"
```

Comment:
If we always import this as `cpool` we might as well rename the package itself to `cpool` 🤷

Reply:
yea, will do!

Reply:
Fixed in 6708b6d